ci/setup-rust.sh

```diff
@@ -0,0 +1,14 @@
+#!bash
+set -eux
+
+RUST_TOOLCHAIN=${1:-1.62}
+
+TMP_DIR=$(mktemp -d -t rust-install.XXXXXXXXXX)
+
+trap "rm -vrf $TMP_DIR" EXIT INT TERM
```

Comment:
What is this for ?

I have the feeling we want to replace this script by a python one, so that nice context manager can handle cleanup-on-teardown nicely

and if this is to remove the `$TMP_DIR/rust-install.sh`, this is far too cumbersome (and you can do `curl <url> | bash` to execute the script without saving it on disk !)

Reply:
done